packages/web-runtime/themes/owncloud/theme.json

```diff
@@ -20,6 +20,7 @@
         "background-default": "#ffffff",
         "background-highlight": "#edf3fa",
         "background-muted": "#f8f8f8",
+        "background-secondary": "#ECECEC",
```

Comment:
Please keep the hex codes all lowercase for consistency. Also need to update them in the docs.

Reply:
Also, please make sure that that you make similar adjustments to the dark theme color palette.

Reply:
alright